test/functional/cmdLineTests/callsitedbgddrext/playlist.xml

```diff
@@ -91,15 +91,14 @@
 		</variations>
 		<command>$(MKTREE) $(REPORTDIR); \
 	cd $(REPORTDIR); \
-	perl $(JVM_TEST_ROOT)$(D)TestConfig$(D)scripts$(D)tools$(D)sysvcleanup.pl zos ; \
+	perl $(JVM_TEST_ROOT)$(D)TestConfig$(D)scripts$(D)tools$(D)sysvcleanup.pl all ; \
 	$(JAVA_COMMAND) $(JVM_OPTIONS) -Xint \
```

Comment:
You can remove the -Xint from all these tests. It's there for historical reasons when the jit was less reliable, but its no longer required. The tests will run faster.